expo-zustand-styled-components/src/components/Profile/Repositories/Repositories.tsx

```diff
@@ -34,6 +39,12 @@ const Repositories = ({ username }: { username: string }) => {
     });
   }, [username]);
 
+  useEffect(() => {
+    useRepoSortFilter(userRepos, search)
+  }, [search]);
```

Comment:
userRepos should be in the deps []

Reply:
Why should useRepos be in the deps array?

Reply:
it is a variable that can be modified, base on the data returned from the API

Reply:
is useRepo a variable that is reactive?

Reply:
not useRepos but userRepos*

Reply:
Yeah, the bug is fixed.
Thanks. 😸